docs/setting-up/server/network.md

```diff
@@ -0,0 +1,22 @@
+# Network
+
+## Ports
+
+This is a list of ports used by the various components of PMM.
+
+For PMM to work correctly, your system's firewall should allow traffic on these ports.
+
+| Destination   | Component     | Protocols | Port/Port Range | Dicertion | Description
+|---------------|---------------|-----------|-----------------|-----------|---------------------
+| PMM Server    | `pmm-server`  | TCP       |   80            | out       | HTTP server, used for web interface
+| PMM Server    | `pmm-server`  | TCP       |  443            | out       | HTTPS server, used for web interface
+| PMM Server    | `pmm-server`  | TCP       | 7771            | both      | gRPC, used for communication between `pmm-agent`, `pmm-admin`
+| PMM Server    | `pmm-server`  | TCP       | 7772            | out       | HTTP1 server, used for older links like `logs.zip`
+| PMM Server    | `pmm-server`  | TCP       | 7773            | both      | Debugging
+| PMM Client    | `pmm-agent`   | TCP       | 7777            | both      | Default `pmm-agent` listen port
```

Comment:
Are these exposed?  I think they're inside the container only right?  where we do gRPC it's over https so 443